keps/262-ConfigurableFailurePolicy/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Configurable Failure Policy
+kep-number: 262
+authors:
+  - "@danielvegamyhre"
+status: provisional
+creation-date: 2023-08-01
+reviewers:
+  - "@kannon92"
```

Comment:
Maybe @mimowo can weigh in as a reviewer? If possible.

As I see a big overlap between this and `PodFailurePolicy`.

Reply:
Agree, that would be great.